### PR TITLE
Adopt a few abandoned plugins working down the wmi long tail epic

### PR DESCRIPTION
**Pull request by NotMyFault**
I want to adopt a few abandoned plugins to help out with the wmi long tail epic.

### CODE REVIEW

**Comment by MarkEWaite** [comment]
@olivergondza can you confirm that you approve the adoption of the build-alias-setter plugin by @NotMyFault ?  I am confident that the plugin will benefit from his adoption of the plugin.

**Comment by MarkEWaite** [comment]
+1 from me for the HSTS filter plugin and the scoring load balancer plugin.

**Comment by NotMyFault** [comment]
The plugin is labeled as up for adoption Von meinem iPhone gesendetAm 21.12.2022 um 00:44 schrieb Mark Waite ***@***.***>:﻿
@olivergondza can you confirm that you approve the adoption of the build-alias-setter plugin by @NotMyFault ?  I am confident that the plugin will benefit from his adoption of the plugin.

**Comment by NotMyFault** [comment]
> @olivergondza can you confirm that you approve the adoption of the build-alias-setter plugin by @NotMyFault ?  I am confident that the plugin will benefit from his adoption of the plugin.

The plugin is labeled as up for adoption.

**Review by timja — APPROVED** [review]

File: permissions/plugin-hsts-filter-plugin.yml [hunk 1]
```diff
@@ -5,4 +5,5 @@ issues:
   - jira: '15944'  # hsts-filter-plugin
 paths:
   - "org/jenkins-ci/plugins/hsts-filter-plugin"
-developers: []
+developers:
+  - "notmyfault"
```
Comment (NotMyFault) [inline permissions/plugin-hsts-filter-plugin.yml:9]:
```suggestion
  - "notmyfault"
cd:
  enabled: true
```

File: permissions/plugin-scoring-load-balancer.yml [hunk 2]
```diff
@@ -5,4 +5,5 @@ issues:
   - jira: '18051'  # scoring-load-balancer-plugin
 paths:
   - "jp/ikedam/jenkins/plugins/scoring-load-balancer"
-developers: []
+developers:
+  - "notmyfault"
```
Comment (NotMyFault) [inline permissions/plugin-scoring-load-balancer.yml:9]:
```suggestion
  - "notmyfault"
cd:
  enabled: true
```

File: permissions/plugin-build-alias-setter.yml [hunk 3]
```diff
@@ -7,3 +7,4 @@ paths:
   - "org/jenkins-ci/plugins/build-alias-setter"
 developers:
   - "olivergondza"
+  - "notmyfault"
```
Comment (NotMyFault) [inline permissions/plugin-build-alias-setter.yml:10]:
```suggestion
  - "notmyfault"
cd:
  enabled: true
```

**Pull request merged**